app/assets/javascripts/discourse/tests/acceptance/user-status-test.js

```diff
@@ -118,6 +123,12 @@ acceptance("User Status", function (needs) {
       userStatus,
       "status description is shown"
     );
+    assert.equal(
+      query(".date-picker").value,
+      "2100-02-01",
+      "ends_at date is shown"
+    );
+    assert.equal(query(".time-input").value, "09:35", "ends_at time is shown");
```

Comment:
One general note about acceptance test is that we're trying to describe behaviour from a user's perspective and from a user's perspective, I don't think `ends_at` will mean much to them. I would recommend using a description which describes how the user will interact with the app.

Reply:
Makes a lot of sense, fixed.